src/downloader/DownloadLinks.java

```diff
@@ -0,0 +1,19 @@
+package downloader;
+
+/**
+ * This class is used to store the web links to the trace archives
+ * @author Alex
+ *
+ */
+public class DownloadLinks {
+	
+	/* Link for the beijing traces */
+	public static String BEIJING_MOBIWAY = 			"http://mobiway.hpc.pub.ro/beijing.zip";
```

Comment:
It's okay to have a default value for these strings, but when you do an instance of Downloader, you should use the configuration files for the initialisation value. User should not modify the java files.

Reply:
The default values are still there but I added a new file witch contains the mode of download (mobiway/dropbox) and then the links for the mobiway. When the user runs the simulator that file will be read and it updates the default links. So if the user wishes to change the mode or the links for the traces all he has to do is to edit configureDownload.txt.

Reply:
It's okay these way. Please a suggestive name for configuration file. Maybe, you can name it Download.config

Reply:
Done